client/src/components/AuthMenu/AuthMenu.tsx

```diff
@@ -22,6 +24,12 @@ const AuthMenu = (): JSX.Element => {
     handleClose();
     logout();
   };
+  const handleProfile = () => {
+    history.push('/profiledetails');
+  };
+  const handlemybookings = () => {
+    history.push('/bookings');
+  };
```

Comment:
Repeated code here from #63 

Please be cautious of this so we aren't reviewing code again.